src/SpinWaveTheory/SWTCalculations.jl

```diff
@@ -634,46 +713,50 @@ function intensity_formula(f::Function,swt::SpinWaveTheory,corr_ix::AbstractVect
             phase = exp(-2π*im * dot(q_reshaped, sys.crystal.positions[i]))
             Avec_pref[i] = sqrt_Nm_inv * phase
 
-             # TODO: move form factor into `f`, then delete this rescaling
+            # TODO: move form factor into `f`, then delete this rescaling
```

Comment:
What would be the approach to doing this? Basically pull the loop below (`Fill `intensity` array`) inside of `f` one way or another?